trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -0,0 +1,26 @@
+const api = require('../api');
+
+const createFacilityAmendment = async (req, res) => {
+  const { facilityId } = req.body;
+  const { amendmentId } = await api.createFacilityAmendment(facilityId);
```

Comment:
Should we check if `amendmentId` is not null before returning `200` back?
Same applied to below three functions.